test/EFCore.PG.FunctionalTests/BuiltInDataTypesNpgsqlTest.cs

```diff
@@ -869,7 +869,7 @@ public void Sum_Conversions()
         {
             using var context = CreateContext();
 
-            // PostgreSQL SUM() returns numeric for bigint input, bigint for int/smallint inuts.
+            // PostgreSQL SUM() returns numeric for bigint input, bigint for int/smallint ints.
```

Comment:
Register the typo and sell it to Apple as a business idea.

Reply:
Good one...